openAPI/core-back-internal.yaml

```
@@ -263,6 +263,23 @@ paths:
         passthroughBehavior: "when_no_match"
         type: "aws_proxy"
 
+  /journey/check-existing-identity:
+    get:
+      description: "Check Existing Identity"
+      responses:
+        200:
+          description: "Check Existing Identity response"
+          content:
+            application/json:
+              schema:
+                type: "object"
```

I think this should be returning a journeyResponse each time so this can be the following
```suggestion
                $ref: "#/components/schemas/journeyType"
```

---

Nice spot - done